config/jobs/kubernetes-sigs/sig-windows/release-1.28-windows-presubmits.yaml

```diff
@@ -41,10 +41,10 @@ presubmits:
         resources:
           requests:
             cpu: "2"
-            memory: 2Gi
+            memory: 9Gi
```

Comment:
9gi is probably unnecessarily high and is greater than the core:memory ratio AFAIK, can we try something in-between?

Reply:
actually no, that's 4.5gi/core. n1-highmem-8 is 6.5/core before any system reserved.

it's probably similar in the EKS cluster but I still need to dig into that.

Reply:
This was the values we had previously, I think we can use the monitoring dashboards to figure out if we can lower it after getting a week or two worth of runs in

Reply:
it should be fine, though I really need to go look at what sort of nodes we have in the EKS cluster, I've lost track.